packages/typespec-ts/README.md

```diff
@@ -55,7 +55,7 @@ One can further configure the SDK generated, using the emitter options on `@azur
 ```yaml
 options:
   "@azure-tools/typespec-ts":
-    packageDetails:
+    package-details:
```

Comment:
Please don't merge this pr, we need to a way to mark the old option pattern as deprecation.